db/migrate/20160429211034_add_timestamps_to_message_digests.rb

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) 2015 The Regents of the University of Michigan.
+# All Rights Reserved.
+# Licensed according to the terms of the Revised BSD License
+# See LICENSE.md for details.
+
+class AddTimestampsToMessageDigests < ActiveRecord::Migration
+  def change
+    add_timestamps(:message_digests, null: false)
+  end
```

Comment:
This adds both `created_at` and `updated_at` so active record can apply all it's default behaviors.